<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TinventoryReportViewItemTable</class>
 <widget class="QWidget" name="TinventoryReportViewItemTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>751</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="m_mainLayout">
    <item row="1" column="2">
     <widget class="QPushButton" name="m_printButton">
      <property name="text">
       <string>打印</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="m_saveButton">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QTableWidget" name="m_inventoryReportTableWidget">
      <column>
       <property name="text">
        <string>机台名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>加币信息</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>总加币</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>清币信息</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>总清币</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>投币盈利</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
